torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -402,10 +402,6 @@ bool is_tensor_list_and_append_overloaded(PyObject* obj, std::vector<py::handle>
   for (long idx = 0; idx < size; idx++) {
     PyObject* iobj = tuple ? PyTuple_GET_ITEM(obj, idx) : PyList_GET_ITEM(obj, idx);
     if (!is_tensor_and_append_overloaded(iobj, overloaded_args)) {
-      if (throw_error) {
```

Comment:
get rid of throw_error.

Reply:
it's done :), thanks for pointing it to me